lib/features/login/data/local/account_cache_manager.dart

```diff
@@ -35,8 +35,8 @@ class AccountCacheManager {
         .removeDuplicated()
         .whereNot((account) => account.accountId == newAccountCache.accountId)
         .toList();
+      await _accountCacheClient.clearAllData();
```

Comment:
why do that?

Reply:
In case of newAllAccount empty, you not update the cache.

Reply:
So what effect does it have on the current cache, if we follow the old flow? Please describe more clearly the root cause of this problem.